test/test_kvaser.py

```diff
@@ -8,10 +8,7 @@
 import time
 import logging
 import unittest
-try:
-    from unittest.mock import Mock, patch
-except ImportError:
-    from mock import patch, Mock
```

Comment:
We can remove the `mock` dependency if not already done.

Reply:
Done. As well as the future dependency. Six will be removed separately, I opened an issue for it.